README.md

```diff
@@ -4,7 +4,7 @@ Convert and modify subtitle files with NodeJS.
 
 There are many subtitle tools written in node, but there are none that support all popular subtitle and caption formats. `subtitle-converter` builds off of the work of others (most notably `node-captions`, `node-webvtt`, and `subtitles-parser`) in order to ultimately become the only subtitle module necessary to include in your node project.
 
-Currently supported input file types: `dfxp, scc, srt, ttml, vtt`
+Currently supported input file types: `dfxp, scc, srt, ttml, vtt, ssa, ass`
```

Comment:
SSA is the same as ASS? I don't see support for it it in `lib/parsers/index.js`

Reply:
ASS is the newer version of SSA files that has more formatting features.  Both can be identified with the `assRegex` in `lib/shared/extensionRegex.js` and for our purposes are parsed the same way, so it would be redundant to create a separate SSA regex and parser.